config/initializers/yamls.rb

```diff
@@ -0,0 +1 @@
+NEWS = YAML.load_file(Rails.root.join('config', 'news.yml'))
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.